### PR TITLE
Changes for Conversation Hold/Resume

### PR DESCRIPTION
**Pull request by muhammadzubair07**
Version 1.17.5.0 (On top of 1.17.3.x)

Added CONVERSATION_PAUSED, CONVERSATION_RESUMED in CimEventName class.
Added ewtMinValue, ewtMaxValue in PrecisionQueueEntity.java
Added HoldTimerDetails class.
Added HoldTimerDetails in ConversationDto and Conversation object.
Implemented Serializable in multiple classes.

### CODE REVIEW

File: src/main/java/com/ef/cim/objectmodel/KeycloakUser.java [hunk 1]
```diff
@@ -19,7 +20,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @ToString
-public class KeycloakUser {
```
Comment (ahmad-bappi) [inline src/main/java/com/ef/cim/objectmodel/KeycloakUser.java:22]: What is this needed?

File: src/main/java/com/ef/cim/objectmodel/Team.java [hunk 2]
```diff
@@ -6,12 +6,14 @@
...
 import lombok.ToString;
 
+import java.io.Serializable;
+
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
 @ToString
-public class Team {
```
Comment (ahmad-bappi) [inline src/main/java/com/ef/cim/objectmodel/Team.java:14]: Why is this needed ?

File: src/main/java/com/ef/cim/objectmodel/KeycloakUser.java (hunk 1 above)
Comment (muhammadzubair07) [reply src/main/java/com/ef/cim/objectmodel/KeycloakUser.java:22]: I have used Conversation object as the data for CONVERSATION_HOLD timer, the Conversation object is referring to CCUser which then uses the KeycloakUser, this was giving Serialization error while writing the data to JMS.

File: src/main/java/com/ef/cim/objectmodel/Team.java (hunk 2 above)
Comment (muhammadzubair07) [reply src/main/java/com/ef/cim/objectmodel/Team.java:14]: Same as the reason with KeycloakUser object.
